webpack.test.bootstrap.ts

```diff
@@ -0,0 +1,2 @@
+let testContext = (<{ context?: Function }>require).context('./', true, /\.test\.ts/);
```

Comment:
Not sure what is the error that you are getting on hanging tests, but as long as it is 'require is not define' type of thing we shouldn't need browserify.